ai/Gemini_ai_call_server.py

```
@@ -0,0 +1,846 @@
+        self.websocket = None
+        self._is_first_response = True
+        self._is_first_response_complete = False
+        self.conversation_history = []
+        self._closing = False  # 종료 중인지 추적하는 플래그 추가
+        self._recv_lock = asyncio.Lock()  # recv 호출을 위한 Lock 추가
+
+    async def process_user_input(self, text: str):
+        """사용자 입력을 처리하고 AI 응답을 생성합니다."""
+        try:
+            print(f"[시스템] 사용자 입력 처리 시작: {text}")
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)